pom.xml

```diff
@@ -10,7 +10,7 @@
 
   <artifactId>accurev</artifactId>
   <packaging>hpi</packaging>
-  <version>0.7.16-SNAPSHOT</version>
+  <version>0.7.17</version>
```

Comment:
PLEASE don't change the version number. Don't do internal deploys without snapshot, otherwise you block me from doing further work on that version number

Reply:
Apologies.
Shall change it back to 7.16-SNAPSHOT?

Reply:
Yes, please

Reply:
Done. Pushed.
Are we good now?